bundles/org.openhab.binding.tr064/src/main/java/org/openhab/binding/tr064/internal/soap/SOAPValueConverter.java

```diff
@@ -244,7 +244,7 @@ private State processMacSignalStrength(State state, Tr064ChannelConfig channelCo
     @SuppressWarnings("unused")
     private State processTamListURL(State state, Tr064ChannelConfig channelConfig) throws PostProcessingException {
         try {
-            ContentResponse response = httpClient.newRequest(state.toString()).timeout(timeout, TimeUnit.MILLISECONDS)
+            ContentResponse response = httpClient.newRequest(state.toString()).timeout(timeout, TimeUnit.SECONDS)
```

Comment:
Perhaps the instance variable `timeout` could also be renamed to `timeoutSeconds` or similar to avoid such mistake from being repeated?

Reply:
I don't think that is necessary. It was a refactoring error when the configurable timeouts were introduced in #14468 (the fixed value was in ms).